pCrunch/LICENSE.txt

```diff
@@ -1,201 +0,0 @@
-                                Apache License
```

Comment:
Should pCrunch still have a License as a standalone?

Reply:
Yes!  If it has its own repo, it should have its own license